packages/embed-jsx/src/lib/EmbedJsx.ts

```diff
@@ -0,0 +1,142 @@
+import { MessageEmbed, ColorResolvable } from 'discord.js';
+
+const EMBED_TYPES = ['embed', 'title', 'field', 'timestamp', 'footer', 'description', 'url', 'image'] as const;
```

Comment:
author and thumbnail are missing

Reply:
I have no idea how to do author cause of all the options, suggestions welcome

Reply:
```jsx
<author iconURL="https://owo.cat">
Hi MOM!
</author>
```

```jsx
<Thumbnail url="oof" />
```

Reply:
Fixed in e893298837191db906af815efd1562aa8a5e6ae6